.pre-commit-config.yaml

```diff
@@ -84,3 +84,12 @@ repos:
         args:
         -   "--req client/package.json"
         -   "--lock client/package-lock.json"
+-   repo: https://github.com/EpicPi/ReqLockHook
```

Comment:
Ensures that you update the version if you update the Dockerfile.

Reply:
So `DEVBOX_VERSION` is akin to `package.json`'s `package-lock.json` for us? Does it get auto-generated or would we have to manually increase the version anytime we change the dockerfile

Reply:
Manual increase. I just reused the gh action here because it did almost the exact thing we needed.

Reply:
sgtm